generated-docs/out/backends/summary.md

```diff
@@ -31,6 +31,12 @@ Class                                Effect type                      Supported
 ``AsyncHttpClientMonixBackend``      ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  no
 ``AsyncHttpClientCatsBackend``       ``F[_]: cats.effect.Concurrent`` n/a                                               no                         no
 ``AsyncHttpClientFs2Backend``        ``F[_]: cats.effect.Concurrent`` ``fs2.Stream[F, Byte]``                           yes (regular & streaming)  no
+``ArmeiraFutureBackend``             ``scala.concurrent.Future``      n/a                                               yes (regular)              yes (expect for InputStream)
```

Comment:
good point about `InputStreams`, though that's the case for all backends - they are blocking in their nature so it's not possible to handle them in a  non-blocking way, so I'd leave this out